SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialMetricsDisplay.cpp

```diff
@@ -147,8 +153,12 @@ void ASpatialMetricsDisplay::DrawDebug(class UCanvas* Canvas, APlayerController*
 	}
 }
 
-void ASpatialMetricsDisplay::ToggleStatDisplay()
+
+void ASpatialMetricsDisplay::SpatialToggleStatDisplay()
 {
+#if !UE_BUILD_SHIPPING
+	check(Role != ROLE_Authority);
```

Comment:
Does this check infer something will go wrong if it fails? Or just that it's strange to call it on a server? I'd either remove it, or switch it to check for client instead of non-authoritative.

Reply:
Will change to checking netmode against NM_Client.  It serves both as a useful guard against calling in the wrong context, and as a hint to programmers about how the class operates.